### PR TITLE
Doc tweaks

### PR DESCRIPTION
**Pull request by nebrius**
I updated the roles and responsibilities guide to link to the Slack inviter, and added a decision criteria to the admissions policy that used to be unofficial.

### CODE REVIEW

File: docs/roles-responsibilities.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ Contributor are expected to follow the [Code of Conduct](https://github.com/node
 
 Contributors have the following level of access and permissions for the working group.
 
-- Have access to public slack channel. This does not currently exist, but will be created some time in the near future.
+- Have access to public slack channel
+    - You can invite yourself [here](http://node-inclusivity.nebri.us/)
```
Comment (scottgonzalez) [inline docs/roles-responsibilities.md]: All the other bullet points end in periods. Can you update these?

**Comment by scottgonzalez** [comment]
One minor comment, but other than that this looks good to merge.

File: docs/roles-responsibilities.md (hunk 1 above)
Comment (nebrius) [inline docs/roles-responsibilities.md]: Good catch, will fix

**Comment by nebrius** [comment]
Fixed. FYI I also squashed commits, so GitHub is showing things a _little_ funky with regards to history

**Comment by tentakel** [comment]
LGTM!

**Pull request merged**